### PR TITLE
Docs: Polished Trigger and Plando guides

### PR DESCRIPTION
**Pull request by recklesscoder**
## What is this fixing or adding?
A pass of polish for the trigger and plando guides.

The changes in roughly descending order of severity are:
- Hopefully improved the clarity of the trigger part explanation
- Fixed the last trigger example (indents were messed up)
- Minor nomenclature changes, that someone should review in context
    - "root category" -> "category"
    - "root option" -> "category"
    - "imaginary names" -> "imaginary values"
- Moved some links to be inline
- Added commas
- Fixed some capitalization
- Misc homogenization

### CODE REVIEW

File: worlds/generic/docs/plando_en.md
Comment (SoldierofOrder) [inline worlds/generic/docs/plando_en.md:14]:
```suggestion
On the website, plando will already be enabled. If you will be generating the game locally, plando features must be
```

File: worlds/generic/docs/plando_en.md
Comment (SoldierofOrder) [inline worlds/generic/docs/plando_en.md:150]:
```suggestion
4. This block has an 80% chance of occurring, and when it does, it will place all but 1 of the items randomly among the
```

File: worlds/generic/docs/plando_en.md
Comment (SoldierofOrder) [inline worlds/generic/docs/plando_en.md:209]:
```suggestion
1. These connections are decoupled, so going into the Lake Hylia Cave Shop will take you to the inside of Cave 45, and
```

File: worlds/generic/docs/plando_en.md
Comment (SoldierofOrder) [inline worlds/generic/docs/plando_en.md:210]:
```suggestion
   when you leave the interior, you will exit to the Cave 45 ledge. Going into the Cave 45 entrance will then take you to
```

File: worlds/generic/docs/plando_en.md
Comment (SoldierofOrder) [inline worlds/generic/docs/plando_en.md:212]:
```suggestion
   you to their locations as normal, but leaving Old Man Cave will exit at Agahnim's Tower.
```

**Review by SoldierofOrder — CHANGES_REQUESTED** [review]
Coupla little commas. The changes look good otherwise. :)

**Review by alwaysintreble — APPROVED** [review]
This is all just minor grammar and fixing that indentation in the SoE section that broke at some point which I noticed recently and hadn't gotten around to fixing yet so all of it LGTM after soldieroforder's comments get merged in

**Comment by recklesscoder** [comment]
I found some more crimes against grammar and particularly poorly worded sentences I somehow glossed over the first time.

**Review by black-sliver — APPROVED** [review]
Happy little commas =)

Thanks!

**Pull request merged**
